pageserver/src/config.rs

```diff
@@ -806,6 +845,8 @@ max_file_descriptors = 333
 initial_superuser_name = 'zzzz'
 id = 10
 
+metric_collection_interval = '222 s'
+metric_collection_endpoint = 'http://localhost:80/metrics'
```

Comment:
I think this will become a matter of needing to be fixed on the next deploy, related to us needing to keep changelogs and all that. However it doesn't seem like nothing will fail horribly if this is not fixed right away, just some errors from the line 275...

Nothing on https://github.com/neondatabase/cloud/issues/3336.

Just thinking if this should be a comment line instead not to actually enable it with bogus endpoint.

Reply:
these are just nonsense values in the config parsing unit test

real defaults set above, ee `pub mod defaults ` and `DEFAULT_CONFIG_FILE`

Reply:
Aah, yes, misread, sorry.

It seems that this is not in the default config file even as commented out?